examples/functions/RefinementResponse.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What was refined, I don't see the difference to the passed trip.

Reply:
@herlitze  This was only a test for elements and structure not content. But you are right we should provide a "good" example.

Reply:
now refinement done